docs_src/walk-through/Ch-WalkthroughReading.md

```diff
@@ -2,68 +2,99 @@
-If an origin is not provided, no origin will be set for the Event or
-Reading, however every Event and Reading is provided a Created and
-Modified timestamp in the database to give the data some time context.
+Use either the Postman or Curl tab below to walkthrough sending an `Event` with `Reading`s to core data.
+
+=== "Postman"
+
+    Make a POST request to `http://localhost:48080/api/v1/event` with the body below.
+
+    ``` json
+    {"device":"countcamera1","readings":[{"name":"HumanCount","value":"5"},{"name":"CanineCount","value":"3"}]}
+    ```
+
+    If your API call is successful, you will get a generated ID (a UUID) for your new `Event` as shown in the image below.
+
+
+    ![image](EdgeX_WalkthroughSendEvent.png)
 
-    BODY: {"device":"countcamera1","origin":1471806386919, "readings":[{"name":"humancount","value":"1","origin":1471806386919},{"name":"caninecount","value":"0","origin":1471806386919}]}
+=== "Curl"
 
-**Origin Timestamp Recommendation!**
+    Make a curl POST request as shown below.
 
-Note: Smart devices will often timestamp sensor data and this timestamp
-can be used as the origin timestamp. In cases where the sensor/device is
-unable to provide a timestamp ("dumb" or brownfield sensors), it is
-recommended that the Device Service create a timestamp for the sensor
-data that is applied as the origin timestamp for the Device.
+    ``` shell
+    curl -X POST -d '{"device":"countcamera1","readings":[{"name":"HumanCount","value":"5"},{"name":"CanineCount","value":"3"}]}' localhost:48080/api/v1/event
+    ```
 
-## Reading data
+#### Origin Timestamp
+If desired, the device service can also supply an origin property in the `Event` or `Reading` object to suggest the time (in Epoch
+timestamp/milliseconds format) at which the data was sensed/collected.
+If an origin is not provided, no origin will be set for the `Event` or
+`Reading`.  However, every `Event` and `Reading` is provided a Created and
+Modified timestamp by the database when it is saved to give the data some time context.
+
+``` json
+{"device":"countcamera1","origin":1471806386919, "readings":[{"name":"HumanCount","value":"1","origin":1471806386919},{"name":"CanineCount","value":"0","origin":1471806386919}]}
+```
+
+!!! Note
+    Smart devices will often timestamp sensor data and this timestamp
+    can be used as the origin timestamp. In cases where the sensor/device is
+    unable to provide a timestamp ("dumb" or brownfield sensors), it is
+    recommended that the device service create a timestamp for the sensor
+    data that it be applied as the origin timestamp for the device.
+
+## Exploring Events/Readings
 
-Now that an Event (or two) and associated Readings have been sent to
-Core Data, you can use the Core Data API to explore that data that is
+Now that an `Event` (or two) and associated `Readings` have been sent to
+core data, you can use the core data API to explore that data that is
 now stored in MongoDB.
```

Comment:
Does the `MongoDB` here need to be replaced with other more generic term like `local database`?